src/magento2/harness/attributes/docker.yml

```
@@ -5,6 +5,14 @@ attributes:
         MAGE_MODE: "= (@('app.mode') == 'production' ? 'production' : 'developer')"
       environment_secrets:
         MAGENTO_CRYPT_KEY: = @('magento.crypt.key')
+    mysql:
+      image: "mariadb:10.4"
+    redis:
+      image: "redis:5"
+    redis_session:
+      image: "redis:5"
+    elasticsearch:
+      image: "elasticsearch:7.8.1"
```

Maybe:
```suggestion
      image: "elasticsearch:7.9.1"
```

This is the only 7.x tag listed on https://hub.docker.com/_/elasticsearch but unsure if Adobe Commerce Cloud is running this version

---

7.9 is not yet supported by Magento officially (at least that is what they said)

---

to be fair, they said 7.7 is the latest documented supported version, but they already installed 7.8 for some clients to test it out, they didn't mention 7.9 (maybe it is not yet available in the package repository which they use)